README.md

```diff
@@ -41,6 +41,9 @@ If you're using [Cargo Edit][cargo-edit], you can call:
 
 ## Building
 
+The library can be built by entering `cublas/` and `cublas-sys/`, and issuing a
+`cargo build` within each directory.
+
```

Comment:
It should be enough to just build `cublas` since `cublas-sys` is a dependency of that.